const fs = require('fs')

try{
    if(fs.)
}catch(err){
    console.error(err);
}